tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_cuda11_1_tensorrt8_0

```diff
@@ -0,0 +1,196 @@
+ARG BASEIMAGE=nvcr.io/nvidia/cuda:11.1-cudnn8-devel-centos7
```

Comment:
Let's not check in this 11_1 dockerfile. (notice i had to hack it in lines 180-185 to get it working)
we're going to use 11_4 Dockerfile.

Reply:
removed